modules/miner.go

```diff
@@ -0,0 +1,18 @@
+package modules
+
+import (
+	"github.com/NebulousLabs/Sia/consensus"
+)
+
+// The Miner interface provides access to mining features.
+type Miner interface {
+	// FindBlock will have the miner make 1 attempt to find a solved block. It
+	// will give up after a few seconds, returning a block, a bool indiciating
```

Comment:
FindBlock: finds a block on top of the current consensus set, and submits that block to the consensus package.

SolveBlock: given a block, which doesn't need to be valid, and a target, which doesn't need to be the current target of the consensus set, solve the block. Don't send the block to the consensus package.

SolveBlock is useful for testing, and not much more. FindBlock is ultimately also useful for testing and not much more, but it's a simpler function call and it does more things.